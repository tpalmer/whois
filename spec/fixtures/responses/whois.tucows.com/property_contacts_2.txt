Registrant:
 VistaPrint Technologies Ltd
 Canon's Court
 22 Victoria Street
 Hamilton,  HM12
 BM

 Domain name: LEDVERLICHTINGSOEST.COM


 Administrative Contact:
    Administrator, Domain  csadmin@vistaprint.com
    95 Hayden Avenue
    Lexington, MA 02421
    US
    +1.7816526199    Fax: +1.7816526096

 Technical Contact:
    Administrator, Domain  csadmin@vistaprint.com
    95 Hayden Ave.
    Lexington, MA 02421
    US
    +1.7816526199    Fax: +1.7816526096



 Registration Service Provider:
    Vistaprint, csadmin@vistaprint.com
    866-811-1674



 Registrar of Record: TUCOWS, INC.
 Record last updated on 05-Jun-2012.
 Record expires on 19-Jun-2013.
 Record created on 19-Jun-2008.

 Registrar Domain Name Help Center:
    http://tucowsdomains.com

 Domain servers in listed order:
    NS4.VPWEB.COM   
    NS3.VPWEB.COM   


 Domain status: clientTransferProhibited
                clientUpdateProhibited

The Data in the Tucows Registrar WHOIS database is provided to you by Tucows
for information purposes only, and may be used to assist you in obtaining
information about or related to a domain name's registration record.

Tucows makes this information available "as is," and does not guarantee its
accuracy.

By submitting a WHOIS query, you agree that you will use this data only for
lawful purposes and that, under no circumstances will you use this data to:
a) allow, enable, or otherwise support the transmission by e-mail,
telephone, or facsimile of mass, unsolicited, commercial advertising or
solicitations to entities other than the data recipient's own existing
customers; or (b) enable high volume, automated, electronic processes that
send queries or data to the systems of any Registry Operator or
ICANN-Accredited registrar, except as reasonably necessary to register
domain names or modify existing registrations.

The compilation, repackaging, dissemination or other use of this Data is
expressly prohibited without the prior written consent of Tucows.

Tucows reserves the right to terminate your access to the Tucows WHOIS
database in its sole discretion, including without limitation, for excessive
querying of the WHOIS database or for failure to otherwise abide by this
policy.

Tucows reserves the right to modify these terms at any time.

By submitting this query, you agree to abide by these terms.

NOTE: THE WHOIS DATABASE IS A CONTACT DATABASE ONLY.  LACK OF A DOMAIN
RECORD DOES NOT SIGNIFY DOMAIN AVAILABILITY.


