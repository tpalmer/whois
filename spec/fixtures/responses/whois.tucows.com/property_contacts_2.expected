#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].name         == "Administrator, Domain"
  should: %s[0].organization == nil
  should: %s[0].address      == "95 Hayden Avenue"
  should: %s[0].city         == "Lexington"
  should: %s[0].zip          == "02421"
  should: %s[0].state        == "MA"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "+1.7816526199"
  should: %s[0].fax          == "+1.7816526096"
  should: %s[0].email        == "csadmin@vistaprint.com"
